sdk/master_board_sdk/CMakeLists.txt

```diff
@@ -73,10 +71,14 @@ ADD_LIBRARY(${PROJECT_NAME} SHARED
   ${MASTER_BOARD_SRC}
 )
 TARGET_LINK_LIBRARIES(${PROJECT_NAME} pthread)
+target_include_directories(
+  ${PROJECT_NAME} PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
```

Comment:
I think the root `include` folder is automatically found by CMake, so I don't see the point with `$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>`. Do you ?

If you remove this, does it change anything ?

Reply:
I think it should be ok without it, it is just recommended by ROS2 ament. And it's modern cmake...
If I understood properly, it has better portability properties... I didn't dig too much about it I have to confess.
I do not have strong opinions on this. I can remove it.

Reply:
Ok so actually there is a CMake error if you do not use it:
```
CMake Error in CMakeLists.txt:
  Target "master_board_sdk" INTERFACE_INCLUDE_DIRECTORIES property contains
  path:

    "/home/mnaveau/devel/workspace_ros2/src/master-board/sdk/master_board_sdk/include"

  which is prefixed in the source directory.
```